Sources/Host/Linux/PTrace.cpp

```diff
@@ -316,6 +318,15 @@ ErrorCode PTrace::writeRegisterSet(ProcessThreadId const &ptid, int regSetCode,
 
   return kSuccess;
 }
+
+#if defined(ARCH_X86) || defined(ARCH_X86_64)
+size_t PTrace::computeDebugRegOffset(int idx) {
```

Comment:
I think `computeDebugRegOffset` could be made into a `constexpr` function very easily.